gustaf/spline/create.py

```diff
@@ -0,0 +1,229 @@
+"""gustaf/spline/create.py
+
+Create operations for spline geometries
+"""
+
+
+import itertools
+import logging
+import numpy as np
+
+
+def extrude(spline, axis=None):
```

Comment:
Do we have better name for `axis`? with `axis`, i'd expect a magnitude separately.   For example, `along`, `extrude_vector`, ...